app/views/shared/partials/_notification_banner.html.erb

```diff
@@ -1,6 +1,6 @@
-<% success_class = local_assigns[:success] ? 'govuk-notification-banner--success' : ''
-   header = local_assigns[:success] ? t('generic.success') : t('generic.important')
-   link_to = false unless local_assigns.has_key?(:link_to) %>
+<% success_class = local_assigns[:success] ? "govuk-notification-banner--success" : ""
```

Comment:
Should this be split into 3 lines like the other examples?

Reply:
They could be, but it doesn't really matter.

The reason they haven't been is because none of the lines were parsed as "empty", so there weren't any blank line "failures" that I needed to manually correct.